src/PaymentEscrow.sol

```diff
@@ -123,11 +123,9 @@ contract PaymentEscrow {
         _pullFunds(auth, value, paymentDetailsHash, signature);
 
         // check capture deadline
-        // TODO: unbork tests
-        // AuthorizationState memory authState = _authorizations[paymentDetailsHash];
-        // if (block.timestamp > authState.captureDeadline) {
-        //     revert AfterCaptureDeadline(uint48(block.timestamp), authState.captureDeadline);
-        // }
+        if (block.timestamp > auth.captureDeadline) {
+            revert AfterCaptureDeadline(uint48(block.timestamp), auth.captureDeadline);
```

Comment:
issue was using `authState.captureDeadline` when the `authState` never actually gets set in the charge flow. Instead we make the comparison with the `auth.captureDeadline` from the payment details directly.

Reply:
Ohhhh nice thank you!